### PR TITLE
Routine i18n updates on 30 Aug 2019

### PR DESCRIPTION
**Pull request by BrettDong**
#### Summary
```SUMMARY: I18N "Routine i18n updates on 30 Aug 2019"```

Updated translation template (`lang/po/cataclysm-dda.pot`) and translation texts (`lang/po/<lang>.po`).

### CODE REVIEW

**Comment by AMurkin** [comment]
What's this lang/po/cataclysm-dda.pot-e file?

You need some text line after the summary line to pass `PR validator check`.

File: lang/po/cataclysm-dda.pot [hunk 1]
```diff
@@ -94,7 +94,7 @@ msgstr ""
 msgid "plant fiber"
 msgstr ""
 
-#. ~ Description for plant fiber
+#. ~ Description for plant fibe
```
Comment (Qrox) [inline lang/po/cataclysm-dda.pot]: Why are these comments missing the last character? (Could probably be related to my recent changes to make it output unix line endings on Windows, but I'm not sure how.)

Comment (BrettDong) [reply lang/po/cataclysm-dda.pot]: Probably yes. I removed the following code from `lang/update_pot.sh` and the output `cataclysm-dda.pot` is then correct.

```bash
# convert line endings to unix
if [[ $(uname -s) =~ ^(CYGWIN|MINGW)* ]]
then
    echo "> Converting line endings to Unix"
    if ! sed -i -e 's/\r$//' lang/po/cataclysm-dda.pot
    then
        echo "Line ending conversion failed. Aborting."
        exit 1
    fi
fi
```

**Comment by BrettDong** [comment]
> You need some text line after the summary line to pass PR validator check.

Is the PR text now complying validator rules? How to trigger a re-check?

File: lang/po/cataclysm-dda.pot (hunk 1 above)
Comment (Qrox) [reply lang/po/cataclysm-dda.pot]: That's weird, it was working properly on my Windows 10. Also the regexp is supposed to remove '\r' at line end only... What platform were you running the script on?

Comment (BrettDong) [reply lang/po/cataclysm-dda.pot]: I'm running on macOS.

**Comment by BrettDong** [comment]
Corrected `cataclysm-dad.pot` is pushed now.

**Comment by ZhilkinSerg** [comment]
> 
> 
> > You need some text line after the summary line to pass PR validator check.
> 
> Is the PR text now complying validator rules? How to trigger a re-check?

Only by `push` right now.

**Pull request merged**